index.html

```diff
@@ -0,0 +1,119 @@
+    </header>
+    <main>
+      <section class="top_section_container">
+        <div class="top_section_container_all">
+          <div class="top_section_text">
+            <p class="top_section_text_title">
+              일상의 모든 물건을<br />거래해보세요
+            </p>
+            <a class="top_section_text_button" href="items.html"
+              >구경하러 가기</a
+            >
+          </div>
+          <div class="top_section_image">
+            <img src="img/Img_home_top.png" alt="메인 이미지" />
+          </div>
+        </div>
+      </section>
+
+      <div class="container">
+        <section class="item">
+          <div class="background_box">
+            <div class="item_image">
+              <img src="img/Img_home_01.png" alt="item1" />
+            </div>
+            <div class="item_text">
+              <p class="item_text_type">Hot item</p>
+              <p class="item_text_title">인기 상품을<br />확인해 보세요</p>
+              <p class="item_text_content">
+                가장 HOT한 중고거래 물품을<br />판다 마켓에서 확인해 보세요
+              </p>
+            </div>
+          </div>
+        </section>
+        <section class="item">
+          <div class="background_box_right">
+            <div class="item_text">
+              <p class="item_text_type">Search</p>
+              <p class="item_text_title">
+                구매를 원하는<br />상품을 검색하세요
+              </p>
+              <p class="item_text_content">
+                구매하고 싶은 물품은 검색해서<br />쉽게 찾아보세요
+              </p>
+            </div>
+            <div class="item_image">
+              <img src="img/Img_home_02.png" alt="item1" />
+            </div>
+          </div>
+        </section>
+        <section class="item">
+          <div class="background_box">
+            <div class="item_image">
+              <img src="img/Img_home_03.png" alt="item1" />
+            </div>
+            <div class="item_text">
+              <p class="item_text_type">Regitster</p>
+              <p class="item_text_title">
+                판매를 원하는<br />상품을 등록하세요
+              </p>
+              <p class="item_text_content">
+                어떤 물건이든 판매하고 싶은 상품을<br />쉽게 등록하세요
+              </p>
+            </div>
+          </div>
+        </section>
+      </div>
+
+      <section class="bottom_section_container">
+        <div class="bottom_section_container_all">
+          <div class="bottom_section_text">
+            <p class="bottom_section_text_title">
+              믿을 수 있는<br />판다마켓 중고 거래
+            </p>
+          </div>
+          <div class="bottom_section_image">
+            <img src="img/Img_home_bottom.png" alt="메인 이미지" />
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="footer_left">
+        <p>©codeit - 2024</p>
+      </div>
+      <div class="footer_center">
+        <a class="footer_center_link" href="privacy.html"> Privacy Policy</a>
+        <a class="footer_center_link" href="faq.html">FAQ</a>
```

Comment:
💊 제안
지금 작성하신 경우 현재 파일이 위치한 디렉토리 기준으로 이동됩니다. 예를 들어 www.example.com/ 이라면, www.example.com/items.html로 이동이 되고 www.example.com/privacy.html이라면 www.example.com/privacy.html/faq.html을 찾게 됩니다.
상대경로보다는 절대 경로로 작성하시는 것이 이러한 문제나 유지보수 측면에서도 유리합니다~

```suggestion
        <a class="footer_center_link" href="/faq.html">FAQ</a>
```